test/testdrive/catalog.td

```diff
@@ -189,6 +190,7 @@ materialize
 > SELECT * FROM mz_catalog.mz_databases
 id          database
 -----------------------
+-1          AMBIENT
```

Comment:
Oh, wait, we should expose this database. It's an implementation detail not meant to be visible to users.